### PR TITLE
OCPBUGS-42534: Console-operator's route healthcheck should have longer retry

### PR DESCRIPTION
**Pull request by jhadvig**
Bumping the retry duration for the healthcheck to 5 seconds, but its up for a discussion whats the proper time.
Also adding additional logging around the healthcheck.

/assign @dgoodwin

### CODE REVIEW

File: pkg/console/controllers/healthcheck/controller.go [hunk 1]
```diff
@@ -149,8 +154,14 @@ func (c *HealthCheckController) Sync(ctx context.Context, controllerContext fact
 
 func (c *HealthCheckController) CheckRouteHealth(ctx context.Context, operatorConfig *operatorsv1.Console, route *routev1.Route) (string, error) {
 	var reason string
+	healthCheckBackoff := wait.Backoff{
+		Steps:    5,
+		Duration: 5 * time.Second,
+		Factor:   1.0,
+		Jitter:   0.1,
+	}
```
Comment (dgoodwin) [inline pkg/console/controllers/healthcheck/controller.go:162]: Not certain but I think this is approximately 25 seconds as written, 5 steps delayed by 5s each time plus some jitter/backoff.

Maybe we should start with 10 seconds to see if we can get this gone, so steps 10, duration 1 second?

File: pkg/console/controllers/healthcheck/controller.go [hunk 2]
```diff
@@ -161,7 +172,7 @@ func (c *HealthCheckController) CheckRouteHealth(ctx context.Context, operatorCo
 				url, _, err = routeapihelpers.IngressURI(route, route.Spec.Host)
 				if err != nil {
 					reason = "RouteNotAdmitted"
-					return fmt.Errorf("console route is not admitted")
+					return fmt.Errorf("%s route is not admitted", route.Name)
```
Comment (dgoodwin) [inline pkg/console/controllers/healthcheck/controller.go:175]: Could you add the logging in here before each return? I believe doing it above will only log once after the function returns, and not during the retry loop. Doing so here will show each error we hit and the timestamp, and give us something to search for to see if the problem is still happening if this retry change fixes it and the test start passsing. (which there's a very good chance of)

**Comment by dgoodwin** [comment]
/lgtm

Thanks for this, hopefully this moves the console to green in 4.17 and 4.18 in a couple days. We'll keep monitoring.

**Comment by jhadvig** [comment]
-retest

**Pull request merged**